packages/azure-http-specs/specs/azure/client-generator-core/usage/main.tsp

```diff
@@ -90,11 +91,31 @@ namespace ModelInOperation {
   op modelInReadOnlyProperty(@body body: RoundTripModel): {
     @body body: RoundTripModel;
   };
+
+  @doc("""
+    Serialize the 'OrphanModel' as request body.
+    
+    Expected body parameter: 
+    ```json
+    {
+      "name": "name",
+      "desc": "desc"
+    }
+    ```
+    """)
+  @global.Azure.ClientGenerator.Core.convenientAPI(false)
+  @route("/orphanModelSerializable")
+  @put
+  op orphanModelSerializable(@body body: unknown): NoContentResponse;
 }
 
 @doc("Not used anywhere, but access is override to public so still need to be generated and exported with serialization.")
 @global.Azure.ClientGenerator.Core.usage(global.Azure.ClientGenerator.Core.Usage.input)
 @global.Azure.ClientGenerator.Core.access(global.Azure.ClientGenerator.Core.Access.public)
 model OrphanModel {
+  @global.Azure.ClientGenerator.Core.clientName("modelName")
```

Comment:
i think we do need to have a way to indicate how to serialize an orphan model. it seems weird that i have an orphan model that one property has a client name.

Reply:
`@clientName` appears expected for me. This case maybe a bit uncommon, but I assume it be pretty common that one language want one name, while other language want another.

And the eventgrid lib does exact this with `@clientName`
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/propertyNameOverrideCsharp.tsp
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/propertyNameOverrideJava.tsp

But yes, we may need some way to say default to JSON, either emitter option, option to TCGC, or option to `@service`